### ISSUE TITLE
How do I evaluated props in JSONX

### ISSUE THREAD

**Issue by tsukhu** [repetere/jsonx#601]
Hi ,

 First of all this is a very impressive project and congratulations for that.

Here is what I am trying
```
const HelloWorld = ({name}: {name: string}) => <p>Hello {name}</p>
```

How can I have the "name" prop be passed and evalualated properly
If there is an example , please point me to it.

Thanks

**Comment by yawetse** [comment]
Closing as part of repository maintenance after #1088 landed on main with current dependency updates and passing tests/build. Please open a new issue with a minimal reproduction if this still reproduces on current main or the next published package.
